CHANGELOG.md

```diff
@@ -1,10 +1,11 @@
-## [Next]
+## [next]
 
 - Add parameters `onTapDown` and `onTapUp` on all buttons. ([#795](https://github.com/bdlukaa/fluent_ui/pull/795))
    - **Breaking: if you use the abstract class `BaseButton`, these two parameters are now required** 
 - Add `PasswordBox` widget. ([#795](https://github.com/bdlukaa/fluent_ui/pull/795))
 - Improve example in Navigation/NavigationView in app. ([#796](https://github.com/bdlukaa/fluent_ui/pull/796))
 - Added Tamil language localization. ([#798](https://github.com/bdlukaa/fluent_ui/pull/798))
+- **BREAKING CHANGE** `TextButton` is renamed to `HyperlinkButton` and `ButtonThemeData.textButtonStyle` is renamed to `ButtonThemeData.hyperlinkButtonStyle` ([#802](https://github.com/bdlukaa/fluent_ui/pull/802))
```

Comment:
Maybe mark `TextButton` as deprecated instead of Breaking change ? The breaking change will be in the next major release, not here